src/main.cpp

```diff
@@ -644,6 +667,14 @@ int main(int argc, char* argv[])
     settings.setValue(".mpackage", "MudletPackage");
     settings.setValue("MudletPackage/.", "Mudlet Package");
     settings.setValue("MudletPackage/shell/open/command/.", "mudlet %1");
+    
+    // Register telnet:// protocol handler
+    const QString mudletExe = QCoreApplication::applicationFilePath().replace('/', '\\');
+    settings.setValue("telnet/.", "URL:Telnet Protocol");
+    settings.setValue("telnet/URL Protocol", "");
+    settings.setValue("telnet/DefaultIcon/.", mudletExe + ",1");
+    settings.setValue("telnet/shell/open/command/.", QString("\"%1\" \"%2\"").arg(mudletExe, "%1"));
```

Comment:
Double the `%` to avoid it being taken as a positional argument HERE...
```suggestion
    settings.setValue("telnet/shell/open/command/.", qsl("\"%1\" \"%%1\"").arg(mudletExe));
```